janusgraph-core/src/main/java/org/janusgraph/graphdb/transaction/StandardJanusGraphTx.java

```diff
@@ -320,6 +326,7 @@ public void close() {
                     config.getVertexCacheSize(), effectiveVertexCacheSize, MIN_VERTEX_CACHE_SIZE);
         }
 
+        relTypeCache = new ConcurrentHashMap<>(30);
```

Comment:
If only used by `JanusGraphLazyRelation`, looks like there's no reason to initialize `relTypeCache` if this transaction doesn't have `lazyLoadRelations()` enabled?

Reply:
Fixed